services/narrativelog/values.yaml

```diff
@@ -1,6 +1,8 @@
 # Default values for narrativelog.
```

Comment:
Needs helm-docs comments for all the settings.

Reply:
Done.